src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Fruit> fruits = new HashMap<>();
+
+    public static Map<String, Fruit> getFruits() {
+        return fruits;
+    }
```

Comment:
there's not much sense in having a getter to a public constant, either remove getter or make it private and create all the methods you need to manipulate fruits through this improvised DAO layer, not directly